hazelcast/src/main/java/com/hazelcast/config/EventJournalConfig.java

```diff
@@ -51,7 +51,7 @@
 
     private String mapName;
     private String cacheName;
-    private boolean enabled = true;
+    private boolean enabled = false;
```

Comment:
I had to make if false to use ConfigUtils.getConfig method for `EventJournalConfig.java`
Why we have this enabled by default ?

Reply:
This was introduced for use in jet and the reasoning for enabling it by default was - if you add it to the config, you are intending to use it. You don't need an additional step for enabling it. Same as for `MapConfig` - if you add it, you're using it.
In any case, this isn't going to work as it is not backwards compatible.

Reply:
I will revert the part with EventJournalConfig.

Reply:
The reason I have changed this is a test failure , `ClientMapNearCacheSerializationCountTest`
It seems that EventJournalConfig is added by default to Map. May be I can avoid adding `EventJournalConfig` instead of changing the default.

Reply:
I have decided to revert get*EventJournal methods completely, because behaviour is documented in their javadoc clearly 
```
If there is no default config as well, it will create one and disable the event journal by default.
```

Reply:
Yes, the behaviour is here: https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/config/Config.java#L3119-L3122